recipes/lielab/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import copy, get, rmdir
+import os
+
+required_conan_version = ">=1.52.0"
+
+class LielabConan(ConanFile):
+    name = "lielab"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sandialabs/Lielab"
+    description = "Lielab is a C++ library for numerical Lie-theory: Lie groups," \
+                  " Lie algebras, homogeneous manifolds, and various functions and algorithms" \
+                  " on these spaces."
+    topics = ("Lie-theory", "Lie-group", "Lie-algebra", "numerical", "header-only")
+    package_type = "header-library"
+    license = "MIT"
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    exports_sources = ("CMakeLists.txt",
+                       "LICENSE",
+                       "lielab/*")
+    
+    no_copy_source = True
+
+    def requirements(self):
+        self.requires("eigen/3.4.0")
+
```

Comment:
```suggestion

    @property
    def _min_cppstd(self):
        return 20

    @property
    def _compilers_minimum_version(self):
        return {
            "gcc": "11",
            "clang": "12",
            "apple-clang": "13.1",
            "Visual Studio": "17",
            "msvc": "193",
        }

    def validate(self):
        if self.settings.compiler.cppstd:
            check_min_cppstd(self, self._min_cppstd)
        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support.")

```

Requires C++20: 

https://github.com/sandialabs/Lielab/blob/v0.1.1/CMakeLists.txt#L131